src/redux/books/books.js

```diff
@@ -1,25 +1,66 @@
+import axios from 'axios';
+
 const ADD_BOOK = 'bookStore/books/ADD_BOOK';
 const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+const GET_BOOKS = 'bookStore/books/GET_BOOKS';
+const getUrl = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/ph8zXbF5wn34KcjWkIqO/books/';
 
 const initialState = [];
 
-export const addBook = (payload) => ({
-  type: ADD_BOOK,
-  payload,
-});
+export const addBook = ({ id, title, category }) => async (dispatch) => {
+  const input = await axios.post(getUrl, {
+    item_id: id,
+    title,
+    category,
+  });
+  if (input.data === 'created') {
+    dispatch({
+      type: ADD_BOOK,
+      book: { id, title, category },
+    });
+  }
+};
```

Comment:
- [ ] Adding a book should also reflect on the app's local state. Currently, I have to refresh the page to see the newly added book.

Reply:
Thanks, It's already implemented! now users don't need to refresh the page to see the newly added book.